src/middleware/yarp/fixture-ymanager/YarpFixManager.cpp

```diff
@@ -61,7 +61,7 @@ bool YarpFixManager::setup(int argc, char** argv) {
         enableRestrictedMode();
 
         // load the fixture (application xml)
-        char szAppName[255];
+        char* szAppName = new char [10];
```

Comment:
I don't understand. The actual (pointer) value of `szAppName` seems to be ignored by `KnowledgeBase::addApplication`, and given that it is a pointer passed by value, it is not useful as an output parameter either. Why it is a argument of the `KnowledgeBase::addApplication` method at all?